src/main/java/org/folio/rest/impl/Processor.java

```diff
@@ -236,16 +236,20 @@ private void processSubFieldMapping(JsonObject subFieldMapping, Object[] remembe
     }
   }
 
+  @SuppressWarnings("unchecked")
   private void handleInstanceFields(JsonArray instanceField, int instanceFieldIndex, List<Object[]> arraysOfObjects,
                                     DataField dataField, Object[] rememberComplexObj)
     throws ScriptException, IllegalAccessException, InstantiationException {
 
     JsonObject jObj = instanceField.getJsonObject(instanceFieldIndex);
     JsonArray subFieldsArray = jObj.getJsonArray("subfield");
 
+    // TODO: check whether it is 100% safe to suppress to unchecked warning here
```

Comment:
JsonArray is a List<Object>: https://github.com/eclipse/vert.x/blob/0f5d8a17c27c92287e5b68f791a8809291d69936/src/main/java/io/vertx/core/json/JsonArray.java#L39

You need to do something like this:

Set<String> subFieldsSet = jObj.getJsonArray("subfield").stream()
      .filter(o -> o instanceof String)
      .map(o -> (String) o)
      .collect(Collectors.toCollection(HashSet::new));